### PR TITLE
Add events + Change Monogram

### PR DESCRIPTION
**Pull request by AdrianBor**

### CODE REVIEW

File: src/components/UserMapMarker.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .UserMapMarker-dot {
-  width: 17px;
-  height: 50px;
-  background: url(../img/monogramBlack.png) no-repeat center center;
+  width: 64px;
+  height: 64px;
+  margin-left: -32px;
```
Comment (lukaswelte) [inline src/components/UserMapMarker.css:4]: Why are these negative margins in there?

File: src/reducers/events.json [hunk 2]
```diff
@@ -50,4 +50,46 @@
...
+  {"lat":"48.1721532","lon":"11.5423903","title":"Absolutely cute open air bar with swing for seats","baseline":"In addition you can watch people making efforts...","color":"#45bfff"} ,
+  {"lat":"48.1721568","lon":"11.5428919","title":"Barbecue area surrounding by tennis field","baseline":"Perfect for a birthday pre-party","color":"#45bfff"} ,
+  {"lat":"48.1383705","lon":"11.5446755","title":"Find some spray paint to stay creative!","baseline":"The guy is really cool","color":"#45bfff"} ,
+  {"lat":"48.1425051","lon":"11.5600526","title":"Cool tiny Berlin style bar for cosy discussion","baseline":"First floor, to the end, seat in front of the window!","color":"#45bfff"} ,
+  {"lat":"48.1409506","lon":"11.5768003","title":"Unique small city cinema with auteur cinema","baseline":"Super tiny but really comfortable and old fashion","color":"#45bfff"} ,
+  {"lat":"48.13794","lon":"11.54324","title":"Best home made fresh Dürum in town","baseline":"Mit alles!!!","color":"#45bfff"} ,
+  {"lat":"48.13775","lon":"11.54372","title":"Fantastic breakfast tea house, unique design","baseline":"Super cosy, perfect for a good moment","color":"#45bfff"} ,
```
Comment (lukaswelte) [inline src/reducers/events.json:88]: Nice!

File: src/components/UserMapMarker.css (hunk 1 above)
Comment (AdrianBor) [reply src/components/UserMapMarker.css:4]: @lukaswelte To center the user. For the moment the center (where the user is) was the top left corner.

File: src/reducers/events.json (hunk 2 above)
Comment (AdrianBor) [reply src/reducers/events.json:88]: :) More comming when I will be able to pin :)

File: src/components/UserMapMarker.css (hunk 1 above)
Comment (lukaswelte) [reply src/components/UserMapMarker.css:4]: Ah ok. Thanks :)

**Review by lukaswelte — APPROVED** [review]

**Pull request merged**